frontend-react/src/components/Table/TableFilters.tsx

```diff
@@ -40,60 +44,30 @@ const inclusiveDateString = (originalDate: string) => {
  * and will use the context to get these values.
  */
 function TableFilters({ filterManager, cursorManager }: SubmissionFilterProps) {
-    const FALLBACK_LOCAL_START = "2999-12-31";
-    const FALLBACK_LOCAL_END = "2020-01-01";
-
     /* Local state to hold values before pushing to context. Pushing to context
      * will trigger a re-render due to the API call fetching new data. We have local
      * state to hold these so updates don't render immediately after setting a filter */
-    const [localStartRange, setLocalStartRange] =
-        useState<string>(FALLBACK_LOCAL_START);
-    const [localEndRange, setLocalEndRange] =
-        useState<string>(FALLBACK_LOCAL_END);
+    const [rangeFrom, setRangeFrom] = useState<string>(FALLBACK_FROM);
+    const [rangeTo, setRangeTo] = useState<string>(FALLBACK_TO);
 
     const updateRange = () => {
-        if (localStartRange && localEndRange) {
+        if (rangeFrom && rangeTo) {
             filterManager.updateRange({
                 type: RangeSettingsActionType.RESET,
                 payload: {
-                    start: new Date(
-                        inclusiveDateString(localStartRange)
-                    ).toISOString(),
-                    end: new Date(localEndRange).toISOString(),
-                },
-            });
-            cursorManager.update({
-                type: CursorActionType.RESET,
-                payload: new Date(
-                    inclusiveDateString(localStartRange)
-                ).toISOString(),
-            });
-        } else if (localStartRange && !localEndRange) {
```

Comment:
These old checks shouldn't be necessary since we have fallbacks. There's no way for a user to eliminate the date from a field entirely.